src/asciidocParser.ts

```diff
@@ -157,13 +157,15 @@ export class AsciidocParser {
     attributes['env-vscode'] = ''
 
     const baseDir = AsciidocParser.getBaseDir(doc.fileName)
+    const templateDir = this.getTemplateDir()
     const options: { [key: string]: any } = {
       attributes,
       backend: 'webview-html5',
       extension_registry: registry,
       header_footer: true,
       safe: 'unsafe',
       sourcemap: true,
+      template_dirs: [templateDir],
```

Comment:
we should not add `template_dirs` if the option is undefined (empty)

Reply:
fixed